index.html

```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
+        </div>
+
+        <div class="secondpage">      
+            <div class="workimage">
+                <img src="ImgPlaceholder.png" alt="Project Image">
+            </div>
+            <h2 class="workheading">Multi-Post Stories</h2>
+            <p class="worktext">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <div class="rightsidebar">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
+            <button type="button" class="projectButton">See Project</button>
+        </div>
+
+        <div class="secondpage">       
+            <div class="workimage">
+                <img src="ImgPlaceholder.png" alt="Project Image">
+            </div>
+            <h2 class="workheading">Multi-Post Stories</h2>
+            <p class="worktext">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <div class="thirdparagraph">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
+            <button type="button" class="projectButton">See Project</button>
+        </div>
+
+        <div class="secondpage">     
+            <div class="workimage">
+                <img src="ImgPlaceholder.png" alt="Project Image">
+            </div>
+            <h2 class="workheading">Multi-Post Stories</h2>
+            <p class="worktext">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <div class="fourthparagraph">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
+            <button type="button" class="projectButton">See Project</button>
+        </div>
+
+
+    </div>
```

Comment:
- [ ] Please the container of the section is in a `section` tag, kindly add your container to the `section` tag 🙃 🙃